src/library/Uno.Material/Styles/Controls/v2/MediaPlayerElement.xaml

```diff
@@ -0,0 +1,653 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
```

Comment:
Format XAML using XAML styler + tabs

Reply:
I think we don't have access to that comment. Unless it's just me. But I will help her set up the xaml style if its not done.
![image](https://github.com/unoplatform/Uno.Themes/assets/90481654/419624fa-535d-4641-b810-5f9c943ce027)